py/tests/pathfinder_worker/test_call.py

```diff
@@ -1894,7 +1888,8 @@ def add_casm_definition(
     compiled_class_definition_path: str,
 ):
     with open(compiled_class_definition_path, "rb") as f:
-        contract_definition = f.read()
+        class_definition = f.read()
+        class_definition = zstandard.compress(class_definition, 10)
```

Comment:
I'd remove the compression level from these calls -- using the highest compression level will just make tests slower with no advantage.